src/suites/cts/gpu_test.ts

```diff
@@ -114,6 +114,44 @@ export class GPUTest extends Fixture {
     });
   }
 
+  checkCopyImageBitmapResult(
+    src: GPUBuffer,
+    expected: ArrayBufferView,
+    width: number,
+    height: number,
+    rowPitch: number,
+    bytesPerPixel: number
+  ): void {
```

Comment:
This helper name implies it's specific to copyImageBitmapToTexture tests. Can you either rename it (if it has more general utility), or move it to a standalone helper function in copyImageBitmapToTexture.spec.ts?

Reply:
For current status of CTS, I think copyImageBitmapToTexture is the only one need this. I prefer to move it into copyImageBitmapToTexture.spec.ts

Reply:
Done. Add a helper class